### ISSUE TITLE
32 bit windows LDC build fails

### ISSUE THREAD

**Issue by WebFreak001** [libmir/mir-algorithm#461]
Trying to build 32 bit windows build with mir-algorithm as dependency:

```
    Building mir-algorithm 3.18.4: building configuration [default]
C:\Users\runneradmin\AppData\Local\dub\packages\mir-algorithm-3.18.4\mir-algorithm\source\mir\bignum\internal\phobos_kernel.d(57,23): Error: version `HaveAsmVersion` defined after use
Error ldc2 failed with exit code 1.
```

See https://github.com/Pure-D/serve-d/actions/runs/4528748421/jobs/7975811435

**Comment by veelo** [comment]
Still an issue in release v3.21.0.

**Comment by thewilsonator** [comment]
@veelo are you able to test if this works with the master branch of this repo? [`e888555`](https://github.com/libmir/mir-algorithm/commit/e888555fbbb5a494f14a6639900842d6210cc9da) (Aug 9 2023) made some changes to that line but was done after v3.21.0 (Aug 2 2023). I believe that commit should fix the issue (`version`s are evaluated before `static if`, so that may well lead to definition after use)

@9il either way this will need a new release tag

**Comment by veelo** [comment]
Indeed, a dependency of `"mir-algorithm": "~master"` does work at the moment. Thanks!

**Comment by 9il** [comment]
Thank you all, [v3.22.0](https://github.com/libmir/mir-algorithm/releases/tag/v3.22.0)

**Comment by thewilsonator** [comment]
Fixed in v3.22.0
